stable/redmine/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: redmine
-version: 5.2.1
+version: 5.2.2
```

```suggestion
version: 6.0.0
```

As there has been a major change in the dependency I think we should also update the major version in the parent chart